tools/ollama.go

```diff
@@ -69,7 +102,7 @@ func (*ollama) Install(logger *log.Logger, installPath string) (err error) {
 		return pkgerrors.WithStack(fmt.Errorf("unsupported architecture %s", a))
 	}
 
-	ollamaDownloadURL := "https://ollama.com/download/ollama-linux-" + architectureIdentifier
+	ollamaDownloadURL := "https://ollama.com/download/ollama-linux-" + architectureIdentifier + "?version=" + ollamaVersion
```

Comment:
Not sure if we should move to the github release page with version instead.

Reply:
Also possible. that here is just the official download location that their installer script uses.

Reply:
I guess we can trust more in the uptime of GH directly.